packages/ui/src/scss/_desktop.scss

```diff
@@ -54,10 +57,9 @@ $font-family: -apple-system,
   --font-title: 600 32px/40px #{$font-family};
```

Comment:
I don't think we need a `fonts` mixin in each of the desktop / mobile stylesheets, as the fonts are considered universal and reusable across both platforms

Reply:
nope is two different DS that's the case. And per our discussion is a mixed version not pure AC.